xml/hwbp_registers.xml

```diff
@@ -129,9 +129,9 @@
         <field name="mte" bits="3" access="R/W" reset="0">
             M-mode trigger enable field.
 
-            0: Triggers do not match/fire while the hart is in M-mode.
+            0: Triggers with action=0 do not match/fire while the hart is in M-mode.
 
-            1: Triggers do match/fire while the hart is in M-mode.
+            1: Triggers with action=0 do match/fire while the hart is in M-mode.
```

Comment:
Saying triggers with action=0 fire almost implies that triggers with other actions don't fire.  What do you think about a clarifying statement saying that mte has no effect on triggers with action!=0?

Reply:
How about reverting just the 1 line:
> 1: Triggers do match/fire while the hart is in M-mode.